src/ViewInstance.js

```diff
@@ -308,6 +318,25 @@ class ViewInstance extends React.Component {
       });
   };
 
+  getLinkedDataEditorId = async () => {
+    const {
+      mutator,
+      stripes,
+      selectedInstance: { id, source },
+    } = this.props;
+
+    if (!id || !isMARCSource(source) || !stripes.hasPerm(LINKED_DATA_EDITOR_PERM)) return;
+
+    try {
+      const response = await mutator.linkedDataEditorId.GET(id);
```

Comment:
I have one question.

When user navigate from one instance to the next instance, I guess the same ViewInstance component will be used for rendering the new Instance.

In such case, the "linkedDataEditorId" in the state will contain the linkedDataEditorId of the previously selected instance before the new API call returns(right?). If the user click on the link during that time, the user will be navigated to the edit screen of previously selected instance.

In order to avoid this, should we set the "linkedDataEditorId" in state as null before this line?

Or, am I totally wrong?